src/containers/board/detailBoard/comment/commentcard/commentcard.tsx

```diff
@@ -113,6 +137,49 @@ function CommentCard({ comment, onDeleteSuccess }: CommentCardProps) {
               <p className='text-md text-text-disabled'>
                 {new Date(createdAt).toLocaleDateString()}
               </p>
+              <div className='ml-15 flex flex-row items-center gap-10'>
+                <IconHandLike />
+                <p className='text-text-primary text-md'>100</p>
+                <IconHandLike className='ml-10 transform rotate-180' />
+                <p className='text-text-primary text-md'>6</p>
+              </div>
```

Comment:
각자 따로 묶여있으면 좋을 거 같아요

```suggestion
              <div className='ml-15 flex flex-row items-center gap-10'>
                <div>
                  <IconHandLike />
                  <p className='text-text-primary text-md'>100</p>
                </div>
                <div>
                  <IconHandLike className='ml-10 transform rotate-180' />
                  <p className='text-text-primary text-md'>6</p>
                </div>
              </div>
```

그리고 이모지 남기는 부분은 각 컴포넌트로 있는게 좋을 거 같네요 
prop으로 이모지 타입을 받아서 top 이면 따봉 bottom이면 아래따봉